build.gradle.kts

```diff
@@ -52,11 +54,21 @@ dependencies {
     // For BSON file serialisation
     implementation("org.mongodb:bson:4.11.1")
     // Spigot
-    compileOnly("org.spigotmc:spigot-api:1.20.3-R0.1-SNAPSHOT")
+//    compileOnly("org.spigotmc:spigot-api:1.20.3-R0.1-SNAPSHOT")
+    // Folia
+    compileOnly("dev.folia:folia-api:1.19.4-R0.1-20230527.192927-40@jar")
```

Comment:
Did you actually manage to get this to build with Java 1.8? The github actions pipeline seems to fails to build this